brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/CountingScribeSpanCollectorNotificationsHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.github.kristofa.brave.scribe;
+
+import com.twitter.zipkin.gen.Span;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+class CountingScribeSpanCollectorNotificationsHandler implements ScribeSpanCollectorNotificationsHandler {
+
+    private final AtomicInteger spansRejected = new AtomicInteger();
+    private final AtomicInteger spansSentEvents = new AtomicInteger();
+    private final AtomicInteger sendingFailures = new AtomicInteger();
+    private volatile int lastQueueSize;
+    private volatile int lastBatchSize;
+
+    @Override
+    public void onQueueSizeChanged(int currentQueueSize) {
+        lastQueueSize = currentQueueSize;
+    }
+
+    @Override
+    public void onSpanRejected(Span rejectedSpan) {
+        spansRejected.incrementAndGet();
+    }
+
+    @Override
+    public void onSpansSent(long durationInMillis, int batchSize, boolean failed) {
+        spansSentEvents.incrementAndGet();
+        if (failed) {
+            sendingFailures.incrementAndGet();
```

Comment:
ex. this sends signals of mixed granularity. for example spansRejected is per span and these are ... who knows? I'd prefer signals to be of the same grain, and use atomic addition if we want to increment in bulk